### PR TITLE
Abstracted the way we get absolute path

### PR DESCRIPTION
**Pull request by surajssd**
Added a `staticmethod` in `utils.Utils` named `get_abspath` which abstracts the use of `os.path.join(Utils.getRoot, path)`

### CODE REVIEW

File: atomicapp/utils.py [hunk 1]
```diff
@@ -354,6 +354,10 @@ def getRoot():
         else:
             return "/"
 
+    @staticmethod
+    def get_abspath(path):
+        return os.path.join(Utils.getRoot(), path)
```
Comment (kadel) [inline atomicapp/utils.py]: I think that we should add `.lstrip('/')` to `path` here, as it should be always called with `lstrip('/')`

`getRoot` returns '/host' or '/'
with leading '/' in second argument `os.path.join` cuts first part in path
```
>>> os.path.join("/host","/foo/bar")
'/foo/bar'

>>> os.path.join("/host","foo/bar")
'/host/foo/bar'
```

**Comment by surajssd** [comment]
@kadel done!

**Comment by kadel** [comment]
#dotests

**Comment by kadel** [comment]
#dotestkubernetes

File: atomicapp/utils.py [hunk 2]
```diff
@@ -354,6 +354,10 @@ def getRoot():
         else:
             return "/"
 
+    @staticmethod
+    def get_abspath(path):
+        return os.path.join(Utils.getRoot(), path.lstrip('/'))
```
Comment (dustymabe) [inline atomicapp/utils.py]: Add a comment about what this function is doing if you don't mind.

File: atomicapp/utils.py [hunk 3]
```diff
@@ -354,6 +354,10 @@ def getRoot():
         else:
             return "/"
 
+    @staticmethod
+    def get_abspath(path):
```
Comment (dustymabe) [inline atomicapp/utils.py]: should we name it something like get_real_abspath() ?

**Comment by dustymabe** [comment]
kube test failed due to yum error reaching repos. Trying again.

**Comment by dustymabe** [comment]
#dotestkubernetes

**Comment by surajssd** [comment]
done as you said also rebased!

**Comment by dustymabe** [comment]
#dotests

**Comment by dustymabe** [comment]
LGTM - @cdrage.. do you mind spot checking this before merge?

File: atomicapp/cli/main.py [hunk 4]
```diff
@@ -474,7 +474,7 @@ def run(self):
             if hasattr(args, item) and getattr(args, item) is not None:
                 args.cli_answers[item] = getattr(args, item)
 
-        lock = LockFile(os.path.join(Utils.getRoot(), LOCK_FILE))
+        lock = LockFile(Utils.get_real_abspath(LOCK_FILE))
```
Comment (cdrage) [inline atomicapp/cli/main.py]: I'll have to rebase this on my PR on removing the lockfile, but that's fine :)

**Comment by cdrage** [comment]
Tests pass locally, code looks a-okay! LGTM :+1: Merge when ready

**Comment by dustymabe** [comment]
#dotestopenshift

**Pull request merged**